pkg/cmd/task/start.go

```diff
@@ -207,6 +207,11 @@ func startTask(opt startOptions, args []string) error {
 		if err != nil {
 			return err
 		}
+		if len(trLast.ObjectMeta.GenerateName) > 0 {
+			tr.ObjectMeta.GenerateName = trLast.ObjectMeta.GenerateName
+		} else {
+			tr.ObjectMeta.GenerateName = trLast.ObjectMeta.Name + "-"
```

Comment:
does that mean we alway going to have a generate name with --last ?

Reply:
If user start a task without ```--last```, GenerateName will be set as based on user specifiled name, like this.
```
tkn t start tp
```
GenerateName wil be set as
```
tp-run-
```

This behavior is the same as before the change.
https://github.com/tektoncd/cli/blob/master/pkg/cmd/task/start.go#L198

Reply:
ah gotcha thanks for explaining it to me!

Reply:
Thank you for your review :smile: